staging/src/github.com/openshift/openshift-apiserver/pkg/project/apiserver/registry/projectrequest/delegated/delegated.go

```diff
@@ -208,14 +208,17 @@ func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 		var restMapping *meta.RESTMapping
 		var mappingErr error
 		for i := 0; i < 3; i++ {
-			restMapping, mappingErr = r.restMapper.RESTMapping(toCreate.GroupVersionKind().GroupKind(), toCreate.GroupVersionKind().Version)
-			if mappingErr == nil {
+			restMapping, err = r.restMapper.RESTMapping(toCreate.GroupVersionKind().GroupKind(), toCreate.GroupVersionKind().Version)
+			if err == nil {
 				break
```

Comment:
you keep mappingErr != nil on second retry.

Reply:
fixed